llvm/lib/Analysis/VectorUtils.cpp

```diff
@@ -141,10 +145,13 @@ bool llvm::isVectorIntrinsicWithOverloadTypeAtArg(Intrinsic::ID ID,
   case Intrinsic::fptoui_sat:
   case Intrinsic::lrint:
   case Intrinsic::llrint:
+  case Intrinsic::vp_lrint:
+  case Intrinsic::vp_llrint:
   case Intrinsic::ucmp:
   case Intrinsic::scmp:
     return OpdIdx == -1 || OpdIdx == 0;
```

Comment:
I don't think the EVL operand can be overloaded, should this just be `OpIdx == 0` for vp_lrint/vp_llrint?

Reply:
IIUC, -1 means the return type. The change in this patch is correct, I think.